apps/cookbook/src/app/showcase/label-showcase/label-showcase.component.html

```diff
@@ -0,0 +1,56 @@
+<div class="example">
+  <p>
+    Recommended usage of Kirby-label and the standard label are illustrated in the following
+    examples. Kirby-label should be used in item components and label should be used in form fields.
+  </p>
+  <br />
+
+  <h2>Use kirby-label in item</h2>
+  <p>Use the kirby-label inside the kirby-item to add title and detail</p>
+  <cookbook-example-viewer [html]="itemExample.template">
+    <div class="example-frame">
+      <cookbook-label-example-item #itemExample></cookbook-label-example-item>
+    </div>
+  </cookbook-example-viewer>
+
+  <p>
+    Get more information about the
+    <a href="https://cookbook.kirby.design/#/home/showcase/item">Kirby-label component</a>
+    in item section.
+  </p>
+
+  <h2>Use Label in forms</h2>
+  <p>Use the native label inside the kirby-form</p>
+  <h3>Input</h3>
```

Comment:
Heres a quick scribble with some ideas to what could make the description a bit more educational, to better solve the original issue outlined in #3918.

```suggestion
  <h2>Other label types</h2>
  <p>Despite its naming, the kirby-label component is not to be confused with the <code><label></code> HTML-element, which has an entirely different purpose of associating a label text with a form control.</p>
  <p>In fact, when you are using the form-field element, a native label element is rendered under-the-hood around the supplied kirby components (input, checkbox, radio-group etc.)</p>
    
    <p>
    More information about the advantage of this approach can be found in the 
    <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/label">
      MDN label documentation
    </a>.
  </p>
  <h3>Form Field with label</h3>
```